### PR TITLE
ship relocated test data & unbreak Record rename

### PR DESCRIPTION
**Pull request by mr-c**

### CODE REVIEW

**Comment by mr-c** [comment]
- [ ] Is it mergable?
- [ ] Did it pass the tests?
- [ ] If it introduces new functionality, is it tested?
- [ ] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  and `make doc` output. Use `make format` and manual fixing as needed.
- [ ] Is it documented in the ChangeLog?
- [ ] Was a spellchecker run on the source code and documentation after
  changes were made?

**Comment by mr-c** [comment]
@luizirber @ctb Ready for review & merge

The alias for the old name of `Record` is needed because khmer v1.4 depends on "screed >= 0.8" and thus will break once the next (py3) enabled version of Screed is released. I suggest we remove the alias in the 1.0 release of Screed.

File: doc/dev/release-checklist.rst [hunk 1]
```diff
@@ -83,8 +83,7 @@ Getting Started
         cd ../../testenv2
         source bin/activate
         pip install nose
-        pip install -e
-        git+https://github.com/ged-lab/screed.git@v${new_version}-${rc}#egg=screed
+        pip install -e git+https://github.com/ged-lab/screed.git@v${new_version}-${rc}#egg=screed
```
Comment (ctb) [inline doc/dev/release-checklist.rst]: change to dib-lab, I think

**Comment by ctb** [comment]
LGTM, modulo comment. Please merge when ready.

A bit more background on what you're doing and why, in the PR description, would help me approach these PRs :)

**Comment by mr-c** [comment]
- [ ] remove alias for Record as it is not needed due to khmer v1.4.1 release

**Comment by mr-c** [comment]
I renamed ged-lab to dib-lab everywhere while I was at it.

**Pull request merged**